### ISSUE TITLE
Gitea Push Event: cannot unmarshal struct field .repository.created_at

### ISSUE THREAD

**Issue by kimdre** [go-playground/webhooks#197]
Using Gitea, when the webhook sends a push event, the following error occurs while parsing the event data:

```go
json: cannot unmarshal string into Go struct field .repository.created_at of type int64
```

The mentioned element looks like this:

```json
"created_at": "2024-07-31T22:08:54+02:00",
```

Could it be that a unix/epoch timestamp as an integer is expected here but a datetime string is returned instead?

**Comment by pansachin** [comment]
Yeah, it's expecting a unix epoch time. Example can be seen here https://github.com/go-playground/webhooks/blob/02d182ac47693b5f7263de2ba7bd46607c80aed8/testdata/github/push.json#L134
